doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -46,9 +228,9 @@ The required arguments are relatively self explanatory:
 * end_month
 * end_day
 
-These start and end points are set using the datasets native calendar.
-All six arguments should be given as integers - the named month string
-will not be accepted.
+These start and end points are set using the datasets native calendar. All six
```

Comment:
Add a note that  start_year and end_year can be omitted, as those are filled in automatically from the dataset definition if not specified here. end_year will be the value in the dataset definition + 1

Reply:
Done.